lib/common.js

```diff
@@ -93,24 +94,17 @@ const overrideRequests = function(newRequest) {
       request: overriddenRequest,
       get: overriddenGet,
     }
-
-    module.request = function(options, callback) {
-      // debug('request options:', options);
-      return newRequest(
-        proto,
-        overriddenRequest.bind(module),
+    // https://nodejs.org/api/http.html#http_http_request_url_options_callback
+    module.request = function(input, options, callback) {
+      return newRequest(proto, overriddenRequest.bind(module), [
+        input,
         options,
-        callback
-      )
+        callback,
+      ])
     }
-
-    module.get = function(options, callback) {
-      const req = newRequest(
-        proto,
-        overriddenRequest.bind(module),
-        options,
-        callback
-      )
+    // https://nodejs.org/api/http.html#http_http_get_options_callback
+    module.get = function(input, options, callback) {
+      const req = module.request(input, options, callback)
```

Comment:
Why is this not calling _newRequest_ instead? This breaks the Node API as well

Reply:
This is done to mimic native Node behavior in case other libs are monkey-patching the function. 
Can you provide more details on "This breaks the Node API as well"? Preferably in a [new issue](https://github.com/nock/nock/issues/new?template=01_bug_report.md).

Reply:
I can make a new issue. But writing the details here as well.

New Relic (https://github.com/newrelic/node-newrelic) is overwriting _http.get_ and _http.request_ like you do. But when _http.get_ is calling _http.request_ like here, a call to _http.get_ will call _http.request_ twice when using nock in a New Relic test. 

To prove that this is not native Node behaviour (at least on Node 12):
```js
const http = require('http');

function request() {
  console.log('http.request', arguments);
}

http.request = request;

// our request function is not called
http.get('http://google.com', () => {});
```